src/docs/release-notes/R23/FullNotes.mdx

```diff
@@ -29,5 +32,38 @@
 ### Miscellaneous
 
 -   Bug fixes & minor updates
+-   Added `blui-toolbar-menu` component.
 
 ### Package Summary
+
+#### Icons
+
+-   @brightlayer-ui/icons v1.9.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/icons-svg v1.9.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/icons-mui v3.0.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/mui/CHANGELOG.md))
+-   @brightlayer-ui/react-native-vector-icons v1.5.0 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/rn-vector/CHANGELOG.md))
+
+#### Angular
+
+-   @brightlayer-ui/angular-components v7.0.0 ([Changelog](https://github.com/brightlayer-ui/angular-component-library/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/angular-themes v7.0.0 ([Changelog](https://github.com/brightlayer-ui/angular-themes/blob/master/CHANGELOG.md))
+-   @brightlayer-ui/angular-auth-workflow v3.0.0 ([Changelog](https://github.com/brightlayer-ui/angular-workflows/blob/master/login-workflow/CHANGELOG.md))
+-   @brightlayer-ui/angular-template-authentication v 4.0.0 ([Changelog](https://github.com/brightlayer-ui/angular-cli-templates/blob/master/src/app/auth-workflow/CHANGELOG.md))
```

Comment:
If the purpose of this “package summary” section is, like in previous release notes, to help people update their package to the latest version, then no template should be mentioned given that they are used only in CLIs. Same comment applies to other frameworks.

Reply:
Open to whatever, @joebochill asked for this

Reply:
For the React Template package, I agree with @huayunh ; adopters don't use our template packages directly, they interact with the CLI package.  We shouldn't have to display them in the package releases since they should be hidden to adopters.